make_deb

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+
+##--------------------------------------------------------------------
+## Copyright (c) 2018 Dianomic Systems
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##--------------------------------------------------------------------
+##
+## Author: Ivan Zoratti, Amarendra K Sinha
+##
+
+set -e
+
+GIT_ROOT=`pwd`    # The script must be executed from the root git directory
+architecture="none"
+usage="$(basename "$0") help {x86|arm} [clean|cleanall]
```

Comment:
do we need 2 architectures? if so do we have system info extractor python modules for specific? and importing based on arch check and copy to debian as per that only?